pkg/webui/lib/components/init.js

```diff
@@ -39,7 +39,7 @@ setConfiguration({
 export default class Init extends React.PureComponent {
 
   componentDidMount () {
-    this.props.dispatch({ type: 'INITIALIZE' })
+    this.props.dispatch({ type: 'INITIALIZE_REQUEST' })
```

Comment:
This breaks the oauth login. See https://github.com/TheThingsNetwork/lorawan-stack/blob/feature/800-logic-middlewares/pkg/webui/oauth/store/actions/init.js#L15, `success` or `failure` action is never dispatched.